templates/admin/settings-page.php

```diff
@@ -1,32 +1,62 @@
 <?php
 
+use LevelLevel\VoorDeMensen\Admin\Settings\Menu;
+use LevelLevel\VoorDeMensen\Sync\EventsSync;
+
 if ( ! current_user_can( 'manage_options' ) ) {
 	wp_safe_redirect( admin_url( 'admin.php' ) );
 	exit;
 }
 
 $is_updated = filter_input( INPUT_GET, 'settings-updated', FILTER_VALIDATE_BOOLEAN );
 if ( $is_updated ) {
-	add_settings_error( 'll_vdm_options', 'll-vdm-options-message', __( 'Settings saved', 'll-vdm' ), 'updated' );
+	add_settings_error( Menu::NAME, 'll-vdm-options-updated', __( 'Settings saved', 'll-vdm' ), 'updated' );
 }
 
+$has_quick_synced_events = filter_input( INPUT_GET, 'll_vdm_quick_sync_events', FILTER_VALIDATE_BOOLEAN );
+if ( $has_quick_synced_events ) {
+	add_settings_error( Menu::NAME, 'll-vdm-quick-sync-events', sprintf( __( 'Synced latest %d events', 'll-vdm' ), EventsSync::RECENT_LIMIT ), 'updated' );
+}
 
-settings_errors( 'll_vdm_options' );
+settings_errors( Menu::NAME );
 ?>
 
 <div class="wrap">
 	<h1>
 		<?php echo esc_html( get_admin_page_title() ); ?>
 	</h1>
 	<form action="options.php" method="post">
+		<?php
+			settings_fields( Menu::NAME );
+			do_settings_sections( Menu::NAME );
+			submit_button();
+		?>
+	</form>
+
+	<form action="<?php echo esc_attr( admin_url( 'admin-post.php' ) ); ?>" method="post">
 
-	<?php
-		settings_fields( 'll_vdm_options' );
-		do_settings_sections( 'll_vdm_options' );
-		submit_button();
-	?>
+		<h2>
+			<?php echo esc_html_e( 'Manually sync events', 'll-vdm' ); ?>
+		</h2>
 
+		<p>
+			<?php echo esc_html_e( 'Events are synced automatically every hour', 'll-vdm' ); ?>
```

Comment:
Dit hoeft niet perse?
```suggestion
			<?php echo esc_html_e( 'Events are synced automatically in the background', 'll-vdm' ); ?>
```

Reply:
resolved in https://github.com/level-level/ll-voordemensen/pull/10/commits/2070a20dcd345c9944525d37690d5d25002864b7